hsg-project.xpr

```diff
@@ -5100,11 +5100,11 @@
                         <scenarioAssociation-array>
                             <scenarioAssociation>
                                 <field name="url">
-                                    <String>repos/administrative-timeline/timeline/</String>
+                                    <String>repos/hsg-staging/frus/volumes/frus1958-60v11mSupp.xml</String>
```

Comment:
@WaxCylinderRevival Just to clarify, did you mean for the validation scenario's association to be on a single volume, or all volumes in the `repos/hsg-staging/frus/volumes` directory?

Reply:
Thanks for flagging, Joe!

I just need the FRUS Secondary Review of Dates Only validation scenario preset to be available as an easy-to-select option for hsg-staging and frus repos, but not the default configuration.  Please advise.

Thank you!